<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="lw_pages">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(171, 171, 171);</string>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/newPrefix/tempshyt/&lt;PIL.PpmImagePlugin.PpmImageFile image mode=RGB size=210x357 at 0x7F80B2391D50&gt;out.jpeg</normalon>
        </iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/newPrefix/tempshyt/&lt;PIL.PpmImagePlugin.PpmImageFile image mode=RGB size=244x388 at 0x7F80B22F96C0&gt;out.jpeg</normalon>
        </iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/newPrefix/tempshyt/&lt;PIL.PpmImagePlugin.PpmImageFile image mode=RGB size=210x357 at 0x7F80B2393430&gt;out.jpeg</normalon>
        </iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/newPrefix/tempshyt/&lt;PIL.PpmImagePlugin.PpmImageFile image mode=RGB size=210x357 at 0x7F80B2392890&gt;out.jpeg</normalon>
        </iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/newPrefix/tempshyt/&lt;PIL.PpmImagePlugin.PpmImageFile image mode=RGB size=210x357 at 0x7F80B23933A0&gt;out.jpeg</normalon>
        </iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/newPrefix/tempshyt/&lt;PIL.PpmImagePlugin.PpmImageFile image mode=RGB size=210x357 at 0x7F80B23931F0&gt;out.jpeg</normalon>
        </iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="f_options">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_open">
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_range">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Split by range</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Range from:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_from"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Range to:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_to"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_split_range">
            <property name="text">
             <string>Extract pages</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_select">
         <property name="title">
          <string>Split by selection</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_spli_sel">
            <property name="text">
             <string>Extract pages</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_merge">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Merge all into one file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../QPdfReader/temp.qrc"/>
 </resources>
 <connections/>
</ui>
